src/ZkFold/Cardano/OffChain/Plonkup.hs

```diff
@@ -22,7 +23,7 @@ import           ZkFold.Prelude                                    (log2ceiling)
 
 --------------- Transform Plonk Base to Plonk BuiltinByteString ----------------
 
-type PlonkupN i n = Plonkup i n 1 BLS12_381_G1 BLS12_381_G2 BuiltinByteString
+type PlonkupN i n = Plonkup (U1 :*: U1) i n Par1 BLS12_381_G1 BLS12_381_G2 BuiltinByteString
```

Comment:
Payload argument (assigned `U1 :*: U1` here) depends on the input type actually, so it has to be another parameter `p` in addition to `i` and `n`

Reply:
ok, i fix it